### ISSUE TITLE
Example on different polymorphic relationships

### ISSUE THREAD

**Issue by amirmasoud** [Laravel-Backpack/CRUD#1767]
Is there any example on how to add different polymorphic relationships? I couldn't find any useful example.

**Comment by ziming** [comment]
Hi @amirmasoud are you still stuck on this question?

**Comment by amirmasoud** [comment]
yep

**Comment by ziming** [comment]
Are you adding a field or column? Which field or column type you are trying to use?

**Comment by amirmasoud** [comment]
There was a project almost 1 year ago, It's abandoned and I don't remember, but I think it was an integer.
I'm not sure if keeping this issue alive can help anyone if so I can take a look into the project and provide more details, otherwise, we can close it.

**Comment by tabacitu** [comment]
I totally agree we should add docs for using Backpack with polymorphic relations. Let's move this to the Docs repo, but keep this open until we do add docs for this. Thanks @amirmasoud !
